### PR TITLE
Fix iGPU typos

### PR DESCRIPTION
**Pull request by khronokernel**

### CODE REVIEW

**Comment by dhinakg** [comment]
@midi1996 this is all your fault

File: OpenCore/config-laptop.plist/haswell.md [hunk 1]
```diff
@@ -137,16 +137,16 @@ Generally follow these steps when setting up your iGPU properties. Follow the co
...
+| iGPU | device-id | AAPL,ig-platform-id | Port Count | Total Stolen Memory | Connectors |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| Intel HD Graphics 4400 | 0a160000 | 0c00160a | 3 | 99MB |  LVDSx1 DPx2 |
+| **Intel HD Graphics 5000 <sup>1</sup>** * | 0a260000 | 0500260a | 3 | 52MB |  LVDSx1 DPx2 |
+| **Intel HD Graphics 5000 <sup>2</sup>** * | 0a260000 | 0600260a | 3 | 52MB |  LVDSx1 DPx2 |
+| Intel Iris Graphics 5100 | 0a2e0000 | 08002e0a | 3 | 99MB |  LVDSx1 DPx2 |
+| Intel Iris Pro Graphics 5200 | 0d260000 | 0700260d | 4 | 99MB |  LVDSx1 DPx2 HDMIx1 |
+| Intel Iris Pro Graphics 5200 | 0d260000 | 0900260d | 1 | 99MB |  LVDSx1 |
+| Intel Iris Pro Graphics 5200 | 0d260000 | 0e00260d | 4 | 131MB |  LVDSx1 DPx2 HDMIx1 |
+| Intel Iris Pro Graphics 5200 | 0d260000 | 0f00260d | 1 | 131MB |  LVDSx1 |
```
Comment (dhinakg) [inline OpenCore/config-laptop.plist/haswell.md:149]: wdym lowercase when using uppercase before

File: OpenCore/config-laptop.plist/kaby-lake.md [hunk 2]
```diff
@@ -137,19 +137,19 @@ Generally follow these steps when setting up your iGPU properties. Follow the co
...
+| Intel HD Graphics 620 | 59160000 | 09001659 | 3 | 39MB |  LVDSx1 DPx2 |
+| **Intel HD Graphics 630** * | 591b0000 | 00001b59 | 3 | 39MB |  LVDSx1 DPx2 |
+| Intel HD Graphics 630 | 591b0000 | 06001b59 | 1 | 39MB |  LVDSx1 |
+| Intel HD Graphics 615 | 591e0000 | 00001e59 | 3 | 35MB |  LVDSx1 DPx2 |
+| Intel HD Graphics 615 | 591e0000 | 01001e59 | 3 | 39MB |  LVDSx1 DPx2 |
+| Intel Iris Plus Graphics 640 | 59260000 | 02002659 | 3 | 58MB |  LVDSx1 DPx2 |
+| Intel Iris Plus Graphics 650 | 59270000 | 04002759 | 3 | 58MB |  LVDSx1 DPx2 |
+| Intel Iris Plus Graphics 650 | 59270000 | 09002759 | 3 | 39MB |  LVDSx1 DPx2 |
+| **Intel UHD Graphics 617** * | 87C00000 | 0000C087 | 3 | 35MB |  LVDSx1 DPx2 |
+| Intel UHD Graphics 617 | 87C00000 | 0500C087 | 3 | 58MB |  LVDSx1 DPx2 |
```
Comment (dhinakg) [inline OpenCore/config-laptop.plist/kaby-lake.md:152]: see above

File: OpenCore/config-laptop.plist/skylake.md [hunk 3]
```diff
@@ -135,23 +135,23 @@ Generally follow these steps when setting up your iGPU properties. Follow the co
...
+| Intel HD Graphics 515 | 191e0000 | 00001e19 | 3 | 56MB |  LVDSx1 DPx2 |
+| Intel HD Graphics 515 | 191e0000 | 03001e19 | 3 | 41MB |  LVDSx1 DPx2 |
+| **Intel Iris Graphics 540** * | 19260000 | 00002619 | 3 | 56MB |  LVDSx1 DPx2 |
+| Intel Iris Graphics 540 | 19260000 | 02002619 | 3 | 58MB |  LVDSx1 DPx2 |
+| Intel Iris Graphics 540 | 19260000 | 04002619 | 3 | 35MB |  LVDSx1 DPx2 |
+| Intel Iris Graphics 540 | 19260000 | 07002619 | 3 | 35MB |  LVDSx1 DPx2 |
+| Intel Iris Graphics 550 | 19270000 | 00002719 | 3 | 56MB |  LVDSx1 DPx2 |
+| Intel Iris Graphics 550 | 19270000 | 04002719 | 3 | 58MB |  LVDSx1 DPx2 |
+| Intel Iris Pro Graphics 580 | 193b0000 | 00003b19 | 3 | 56MB |  LVDSx1 DPx2 |
+| Intel Iris Pro Graphics 580 | 193b0000 | 05003b19 | 4 | 35MB |  LVDSx1 DPx3 |
```
Comment (dhinakg) [inline OpenCore/config-laptop.plist/skylake.md:154]: see above

**Comment by midi1996** [comment]
lol

File: OpenCore/config-laptop.plist/haswell.md (hunk 1 above)
Comment (khronokernel) [reply OpenCore/config-laptop.plist/haswell.md]: Trash

File: OpenCore/config-laptop.plist/skylake.md (hunk 3 above)
Comment (khronokernel) [reply OpenCore/config-laptop.plist/skylake.md]: See above

File: OpenCore/config-laptop.plist/kaby-lake.md (hunk 2 above)
Comment (khronokernel) [reply OpenCore/config-laptop.plist/kaby-lake.md]: see below

**Pull request merged**